rdr_service/resource/schemas/participant.py

```diff
@@ -353,14 +354,36 @@ class ParticipantSchema(Schema):
     enrl_core_participant_minus_pm_time = fields.DateTime()
     enrl_core_participant_time = fields.DateTime()
 
-    # TODO: Depreciate fields here that are duplicated by the new enrollment fields above.
+    # Retaining during Goal 1 transition;  may be deprecated after V30/V31 acceptance
     enrollment_status = fields.EnumString(enum=EnrollmentStatusV2)
     enrollment_status_id = fields.EnumInteger(enum=EnrollmentStatusV2)
     enrollment_member = fields.DateTime()
     enrollment_core_ordered = fields.DateTime()
     enrollment_core_stored = fields.DateTime()
     enrollment_core_minus_pm = fields.DateTime()
 
+    # Goal 1 new additions
+    # TODO:  The v2 fields are temporary to do consistency checks for PEO report vs. RDR values
+    enrollment_status_legacy_v2 = fields.EnumString(enum=EnrollmentStatusV2)
+    enrollment_status_legacy_v2_id = fields.EnumInteger(enum=EnrollmentStatusV2)
+    enrollment_status_v3_0 = fields.EnumString(enum=EnrollmentStatusV30)
+    enrollment_status_v3_0_id = fields.EnumInteger(enum=EnrollmentStatusV30)
+    enrollment_status_v3_0_participant_time = fields.DateTime()
+    enrollment_status_v3_0_participant_plus_ehr_time = fields.DateTime()
+    enrollment_status_v3_0_pmb_eligible_time = fields.DateTime()
+    enrollment_status_v3_0_core_minus_pm_time = fields.DateTime()
+    enrollment_status_v3_0_core_time = fields.DateTime()
+
+    enrollment_status_v3_1 = fields.EnumString(enum=EnrollmentStatusV31)
+    enrollment_status_v3_1_id = fields.EnumString(enum=EnrollmentStatusV31)
```

Comment:
Should this be an EnumInteger field instead?

```suggestion
    enrollment_status_v3_1_id = fields.EnumInteger(enum=EnrollmentStatusV31)
```

Reply:
Resolved in latest